src/main/java/com/sumindaa/demo/employeeservice/model/Employee.java

```diff
@@ -3,9 +3,49 @@
 import java.util.Date;
 
 public class Employee {
-int id;
-char firstName;
-char lastName;
-Date dob;
-char gender;
+    private int id;
+    private char firstName;
```

Comment:
Why they are char, char is only for a single character, you should use string.

Reply:
Done